docs/compression_algorithms/CompressWeights.md

```diff
@@ -24,26 +23,201 @@ from nncf import compress_weights
 compressed_model = compress_weights(model)
 ```
 
-- Compress weights to nf4 data type with group size = 128, except first embedding and last linear layers - they are compressed to 8-bit integer data type.
+- Compress weights symmetrically to 4-bit integer data type with group size = 128, except first embedding and last linear layers - they are compressed to 8-bit integer data type.
 
 ```python
 from nncf import compress_weights
 from nncf import CompressWeightsMode
-compressed_model = compress_weights(model, mode=CompressWeightsMode.NF4)
+compressed_model = compress_weights(model, mode=CompressWeightsMode.INT4_SYM)
 ```
 
-- Compress weights of 90% of layers to nf4 with the group size 64, and the rest of layers to 8-bit integer data type.
+- Generally, `INT4_SYM` mode is the fastest mixed-precision mode, but it may lead to a significant accuracy degradation or perplexity increase.
+  Compressing weights asymmetrically (`INT4_ASYM` mode) is the way to increase accuracy, however in turns it slows down inference a bit.
+  If the accuracy or perplexity is still not satisfying, there are 2 more hyper-parameters to tune: `group_size` and `ratio`.
+  Lower group size and less ratio of 4-bit layers usually improve accuracy at the sacrifice of inference speed.
+  Below is the example how to compress weights of 90% of layers to 4-bit integer asymmetrically with the group size 64, and
+  the rest of layers to 8-bit integer data type. The same parametrization is applicable for `INT4_SYM` mode.
 
 ```python
 from nncf import compress_weights
 from nncf import CompressWeightsMode
-compressed_model = compress_weights(model, mode=CompressWeightsMode.NF4, group_size=64, ratio=0.9)
+compressed_model = compress_weights(model, mode=CompressWeightsMode.INT4_ASYM, group_size=64, ratio=0.9)
 ```
 
-##### Limitations
+- `NF4` mode can be considered for improving accuracy, but currently models quantized to nf4 should not be faster models
+  quantized to 8-bit integer on client processors. Some speed up is expected on server processors in the future releases.
+  Here's the example how to compress weights to nf4 data type with group size = 128, except first embedding and last linear layers - they are compressed to 8-bit integer data type.
```

Comment:
"except first embedding and last linear layers - they are compressed to 8-bit integer data type" - it's already explained above for all 4-bit types. Let's remove it.